web/themes/contrib/civictheme/includes/form.inc

```diff
@@ -45,6 +45,13 @@ function civictheme_preprocess_input(array &$variables): void {
   if (isset($variables['attributes']['#civictheme_theme'])) {
     unset($variables['attributes']['#civictheme_theme']);
   }
+
+  // Fix htmlentity in submit button text.
+  $element = $variables['element'];
+
+  if (isset($element['#type']) && $element['#type'] == 'submit' && !empty($variables['attributes']['value'])) {
```

Comment:
what if this is type `reset` or just a button? the solution should take care of all possible cases

Reply:
I have tried raw filter but didn't work.

And it should work for all button. I can review if you think this solution can work.